src/main/java/org/folio/utils/MetadataUtils.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.utils;
+
+import org.folio.okapi.common.OkapiToken;
+import org.folio.okapi.common.XOkapiHeaders;
+import org.folio.rest.jaxrs.model.Metadata;
+
+import java.util.Date;
+import java.util.Map;
+
```

Comment:
@UtilityClass annotation can be added as we have in other places

Reply:
OK